mindmeld/components/dialogue.py

```diff
@@ -120,7 +120,7 @@ def __init__(self, dialogue_state, **kwargs):
         resolved = {}
         for keys in key_kwargs:
             if len(keys) == 2:
-                single, plural = keys
+                single, plural = keys  # pylint: disable=unbalanced-tuple-unpacking
```

Comment:
I realize this is incidental to your changes, but I hadn't realized we were doing this. In L130 we are assuming `single` is a string, and L132 assumes `plural` is an iterable of strings, but the doc string says `has_entity` and `has_entities` are both `(str, list, set)`

Reply:
Ah, good eyes! Update docstring.